integration-cli/docker_cli_ps_test.go

```diff
@@ -815,23 +707,6 @@ func (s *DockerSuite) TestPsShowMounts(c *check.C) {
 	c.Assert(strings.TrimSpace(string(out)), checker.HasLen, 0)
 }
 
-func (s *DockerSuite) TestPsFormatSize(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	runSleepingContainer(c)
-
-	out, _ := dockerCmd(c, "ps", "--format", "table {{.Size}}")
```

Comment:
This test may actually be testing that adding `{{.Size}}` to the format should automatically set the `--size` option; do we have a test covering that in the CLI?

Reply:
We should make sure we have that test yes, I'll update the test in the docker/cli 😉

Reply:
Added in https://github.com/docker/cli/pull/880/files#diff-8b49518538a0aca318c2d28149b90dccR123